src/decorators.js

```diff
@@ -55,10 +55,15 @@ export function customElement(name: string): any {
 /**
 * Decorator: Indicates that the decorated class is a custom attribute.
 * @param name The name of the custom attribute.
-* @param defaultBindingMode The default binding mode to use when the attribute is bound wtih .bind.
+* @param defaultBindingMode The default binding mode to use when the attribute is bound with .bind.
+* @param aliases The array of aliases to associate to the custom attribute.
 */
-export function customAttribute(name: string, defaultBindingMode?: number): any {
+export function customAttribute(name: string, defaultBindingMode?: number, aliases?: string[]): any {
   return function(target) {
+    if (aliases) {
+      target.aliases = aliases;
```

Comment:
We should put that aliases on the behavior. That can be done with the following API:

```javascript
import {HtmlBehaviorResource} from './html-behavior';
import {metadata} from 'aurelia-metadata';

let r = metadata.getOrCreateOwn(metadata.resource, HtmlBehaviorResource, target);
r.aliases = ...
```

@zewa666 Can you make that change?